repo/gromacs/application.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import os
+from ramble.appkit import *
+from ramble.expander import Expander
+
+
+class Gromacs(SpackApplication):
+    '''Define a Gromacs application'''
+    name = 'gromacs'
+
+    maintainers('douglasjacobsen')
+
+    tags('molecular-dynamics')
+
+    default_compiler('gcc9', spack_spec='gcc@9.3.0')
+    software_spec('impi2018', spack_spec='intel-mpi@2018.4.274')
+    software_spec('gromacs', spack_spec='gromacs@2020.5', compiler='gcc9')
```

Comment:
Let's use the latest release, I suggest:

```suggestion
    software_spec('gromacs', spack_spec='gromacs@2023.3', compiler='gcc12')
```